concourse/pipeline.yml

```diff
@@ -76,6 +76,69 @@ jobs:
         file: git-master/concourse/tasks/test.yml
     on_failure: *slack_alert_on_failure
 
+  - name: deploy-to-sandbox
+    serial: true
+    plan:
+      - get: git-master
+        trigger: true
+        passed: [test]
```

Comment:
Deliberate decision to run sandbox & staging deployments in parallel, with prod following on from staging? Can't see any particular issue with this, just checking.

Reply:
with serial this will make it one after the other ? i think sandpit should be allowed to run in parallel

Reply:
Yeah. Sandbox is supposed to have a different set of behaviours so shouldn't hold back staging/prod.